exercises/pig-latin/example.tt

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'pig_latin'
+
+# Common test data version: <%= abbreviated_commit_hash %>
+class PigLatinTest < Minitest::Test<% test_cases.each do |test_case| %>
```

Comment:
Blech. ERB. Adding a newline here causes formatting problems in the generated test file.